backend/internal/storage/postgres/schema/user.go

```diff
@@ -102,6 +102,45 @@ func (r *UserRepository) UnFollow(ctx context.Context, follower uuid.UUID, follo
 	return true, nil
 }
 
+func (r *UserRepository) CalculateScore(ctx context.Context, id uuid.UUID) (int, error) {
+
+	// Coalesce returns first non-null value in the set of arguments, so if SUM returns null, 0 is defaulted to.
+	// urv, rec, and u are aliases to user_review_vote, recommendation, and user tables.
+	query := `
+    SELECT 
+        COALESCE((
+            SELECT 
+                SUM(CASE WHEN urv.upvote = TRUE THEN 1 ELSE -1 END)
+            FROM 
+                user_review_vote urv
+            WHERE 
+                urv.user_id = u.id
+        ), 0) + 
+        COALESCE((
+            SELECT 
+                SUM(CASE WHEN rec.reaction = TRUE THEN 1 ELSE 0 END)
+            FROM 
+                recommendation rec
+            WHERE 
+                rec.recommender_id = u.id
+        ), 0) AS score
+    FROM 
```

Comment:
we can actually cut out the user table to simplify this since all we use is the u.id value, which is the same as our parameter $1. so, you can remove the `FROM` and `WHERE` clauses and replace the u.id instances with $1

Reply:
Updated!